extern/boostd-data/couchbase/db.go

```diff
@@ -300,3 +300,76 @@ func has(list []cid.Cid, v cid.Cid) bool {
 	}
 	return false
 }
+
+// RemoveAllRecords
+func (db *DB) RemoveAllRecords(ctx context.Context, cursor uint64) error {
```

Comment:
This method should not be exported I think, as it is using an internal cursor.

Reply:
Fixed